internal/gcsx/client_readers/range_reader_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcsx
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/fake"
+	testutil "github.com/googlecloudplatform/gcsfuse/v2/internal/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+const fakeHandleData = "fake-handle"
+
+type rangeReaderTest struct {
+	suite.Suite
+	rangeReader *RangeReader
+}
+
+func TestRangeReaderTestSuite(t *testing.T) {
+	suite.Run(t, new(rangeReaderTest))
+}
+
+func (t *rangeReaderTest) SetupTest() {
+	t.rangeReader = NewRangeReader()
+}
+
+func (t *rangeReaderTest) TearDown() {
+}
+
+func getReadCloser(content []byte) io.ReadCloser {
+	r := bytes.NewReader(content)
+	rc := io.NopCloser(r)
+	return rc
+}
+
+func getReader() *fake.FakeReader {
+	testContent := testutil.GenerateRandomBytes(2)
```

Comment:
Please make it constants or take an input size of the reader.

Reply:
It's not getting use anywhere else, I'll make it const if it will get used at some other place also.